### PR TITLE
find all assumptions for sympy versions (ver2) - fully-automatic

### PR DESCRIPTION
**Pull request by latot**
Hi again, here i'm dealing with this little problem, basically octsympy only have specific assumptions to can use independent of the sympy version, and checking the docs of sympy it doesn't says what assumptions are supported for version, only in the dev:http://docs.sympy.org/dev/modules/core.html

So this code use the dev assumptions and detect which are available in the actual version of Sympy, actually i can't found a way to completely auto-detect (that's why ver1) it but if we update this list when Sympy bring new versions should works fine.

Edited:

Now is fully automated!

Cya.

### CODE REVIEW

File: inst/assumptions.m [hunk 1]
```diff
@@ -66,7 +66,7 @@
 %% @group
 %% A = assumptions('possible');
 %% sprintf('%s ', A@{:@})
-%%   @result{} ans = real positive negative integer even odd rational finite
+%%   @result{} ans = ...
```
Comment (latot) [inline inst/assumptions.m]: this now change version by version.

**Comment by cbm755** [comment]
This is pretty cool!  So `syms p prime` and `isprime(p)` would work right?

Slightly uncomfortable with using the private `_assume_defined` Python object but if that is the best way to get the list of assumptions, then ok.  I wonder why I wanted to whitelist them... let me sleep on this before I merge.

Can you add a line to NEWS?  Maybe something like: "The full set of assumptions implemented by Sympy can now be used".

**Comment by latot** [comment]
Yay! it works:
```
>> syms a prime
>> isprime(a)
ans =  1
```
Adding info to NEWS.

Good Night.

**Comment by cbm755** [comment]
Ok, pulled the trigger.

Its possible this will screw up for someone who wants:
````
syms q prime
````
to give two symbols named `q` and `prime`.  But meh :)

**Pull request merged**